README.md

```diff
@@ -28,25 +105,124 @@ This repository contains extension code for SQL Server input and output bindings
     dotnet add package Microsoft.Azure.WebJobs.Extensions.Sql --version 1.0.0-preview1
```

Comment:
Should we point people to 1.0.0-preview2?

Reply:
What is the difference between preview1 and preview2? @anthonychu probably has a better idea of this.

Reply:
preview2 is the package I uploaded that had the trigger binding.

Reply:
Oh ok. That makes sense. I missed that I wasn't using preview2. I was still using preview1 as I was following the README and thought it was updated. I just tried it, and trigger works. That is definitely a lot simpler. Thanks for pointing this out Chris. I'm going to close the PR for now and update the docs to reflect using preview2.